spec/views/catalog/_document_split.html.erb_spec.rb

```diff
@@ -1,5 +1,5 @@
 require 'spec_helper'
 
-describe 'catalog/_document_split.html.erb' do
-  # TODO js plugin refactor really needs to be redone so that the views can be testable
+describe 'catalog/_document_split.html.erb' do # rubocop:disable RSpec/DescribeClass
```

Comment:
Seems like we might run into this with any and all view specs. Could we add this rule in our `.rubocop.yml` file instead?